apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -158,7 +161,7 @@ function ContextRow ({
             // (selected || bottomItem) && 'text-foreground'
           )}
         >
-          {context?.name}
+          {context?.name || 'No Name'}
```

Comment:
you think its better to show No Name? what if there is a flash of a moment where the name isnt loaded yet? id rather have it be empty than flash No Name